common/djangoapps/student/admin.py

```diff
@@ -1,11 +1,31 @@
 '''
 django admin pages for courseware model
 '''
+from django import forms
 from config_models.admin import ConfigurationModelAdmin
 
 from student.models import UserProfile, UserTestGroup, CourseEnrollmentAllowed, DashboardConfiguration
-from student.models import CourseEnrollment, Registration, PendingNameChange, CourseAccessRole, CourseAccessRoleAdmin
+from student.models import CourseEnrollment, Registration, PendingNameChange, CourseAccessRole
 from ratelimitbackend import admin
+from student.roles import REGISTERED_ACCESS_ROLES
+
+
+class CourseAccessRoleForm(forms.ModelForm):
+    """Form for adding new Course Access Roles view the Django Admin Panel."""
+    class Meta:
+        model = CourseAccessRole
+
+    COURSE_ACCESS_ROLES = [(role_name, role_name) for role_name in REGISTERED_ACCESS_ROLES.keys()]
+    role = forms.ChoiceField(choices=COURSE_ACCESS_ROLES)
```

Comment:
Though do you mean, you could have multiple choice as a field, and save three rows from one action in the Admin Panel?